java/runtime-native/src/main/java/org/ray/runner/RunManager.java

```diff
@@ -623,9 +623,11 @@ private void startRaylet(String storeName, AddressInfo info, int numWorkers,
 
     String resourceArgument = ResourceUtil.getResourcesStringFromMap(staticResources);
 
+    int maximumStartupConcurrency = staticResources.get("CPU").intValue();
+
     // The second-last arugment is the worker command for Python, not needed for Java.
     String[] cmds = new String[]{filePath, rayletSocketName, storeName, ip, gcsIp,
-        gcsPort, "" + numWorkers, resourceArgument,
+        gcsPort, "" + numWorkers, "" + maximumStartupConcurrency, resourceArgument,
```

Comment:
String.valueOf(maximumStartupConcurrency);

Reply:
Thanks! I'll also change it for `"" + numWorkers` then.

Reply:
Fixed.